pixel-point/pixelpoint-website
# feat: add Configu case study

**Describe what changes this pull request brings**
This pull request adds update to Configu case study.

[Design](https://www.figma.com/file/uRyIVXOk6qD7ufNv6JSQRf/Pixel-Point-Website-Design?node-id=5012%3A102453&t=OySnzjQQu0Tlkxvu-4)
[Content](https://www.notion.so/pixelpoint/Configu-case-study-f939b524a48d461f8f7e719b72cb0d19)

**Steps to test**

1. Open the [page](https://pixelpointwebsite-configucasestudy.gatsbyjs.io/case-studies/)
2. Confirm that everything looks and works as expected.

**Screenshots**
Configu case study

<details>
<img width="1417" alt="Screenshot 2022-12-19 at 18 44 45" src="https://user-images.githubusercontent.com/75138180/208439850-d956b147-1aef-46bb-9a27-ca52ab69e0f5.png">

<img width="1089" alt="Screenshot 2022-12-19 at 18 46 55" src="https://user-images.githubusercontent.com/75138180/208440132-fd7804ce-9648-49ca-8f19-bab62ee62a87.png">


</details>
